p2p/ipld/racedetector/is_active.go

```diff
@@ -0,0 +1,7 @@
+package racedetector
+
+var raceDetectorActive = false
+
+func IsActive() bool {
+	return raceDetectorActive
+}
```

Comment:
why moving it to it's own package makes the CI happy, idk.

Reply:
🤷🏻‍♂️